### PR TITLE
Added LOLBAS

### PR DESCRIPTION
**Pull request by osospeed**

### CODE REVIEW

File: packages/LOLBAS/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+pkgdesc='Living Off The Land Binaries And Scripts - (LOLBins and LOLScripts)'
+groups=('blackarch' 'blackarch-windows' )
+arch=('any')
+url='https://github.com/api0cradle/LOLBAS'
+depends=()
+source=('git+https://github.com/api0cradle/LOLBAS.git')
+sha1sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/$pkgname"
```
Comment (PedroSFreitas) [inline packages/LOLBAS/PKGBUILD]: We don't use `pkgname` variable inside the script for a few reasons.
I suggest you to change all `pkgname` to `lolbas` (not capital).

File: packages/LOLBAS/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```
Comment (PedroSFreitas) [inline packages/LOLBAS/PKGBUILD]: Insert a new line above.

File: packages/LOLBAS/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+package() {
+  cd "$srcdir/$pkgname"
+  mkdir -p "$pkgdir/usr/share/windows/$pkgname/"
```
Comment (PedroSFreitas) [inline packages/LOLBAS/PKGBUILD]: - Insert a new line above; and
- we don't use `pkgname` variable inside the script for a few reasons. I suggest you to change all `pkgname` to `lolbas` (not capital).

File: packages/LOLBAS/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname='LOLBAS'
```
Comment (PedroSFreitas) [inline packages/LOLBAS/PKGBUILD]: Change the name to `lolbas` (not capital). If you have it all in capital, the package name will be in capital.

File: packages/LOLBAS/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+pkgver=156.f549508
+pkgrel=1
+pkgdesc='Living Off The Land Binaries And Scripts - (LOLBins and LOLScripts)'
+groups=('blackarch' 'blackarch-windows' )
```
Comment (PedroSFreitas) [inline packages/LOLBAS/PKGBUILD]: remove white space ` )`

**Comment by PedroSFreitas** [comment]
@osospeed thanks.

@noptrix also looks good.

**Comment by noptrix** [comment]
thanks a lot to both

**Pull request merged**